src/main/java/klee/task/Event.java

```diff
@@ -0,0 +1,79 @@
+package klee.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event with a duration.
+ */
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+
+    /**
+     * Constructor for Event class.
+     *
+     * @param description
+     * @param from
+     * @param to
+     */
+    public Event(String description, LocalDateTime from, LocalDateTime to) {
+        super(description);
+        this.type = "E";
+        this.from = from;
+        this.to = to;
+    }
+
+    /**
+     * Returns the type and current completion status of event with its start and end date.
+     *
+     * @return String to represent current state of class.
+     */
+    public String getStatus() {
+        String statusIcon = (isDone ? "X" : " ");
+        return "[" + type + "][" + statusIcon + "] " + description + " (from: "
+                + from.format(DateTimeFormatter.ofPattern("MMM dd yyyy h:ma")) + " to: "
+                + to.format(DateTimeFormatter.ofPattern("MMM dd yyyy h:ma")) + ")";
+    }
+
+    /**
+     * Creates a String to represent this task in the txt file.
+     *
+     * @return String to be stored in txt file.
+     */
+    public String toText() {
+        return super.toText() + " / " + from.format(DateTimeFormatter.ofPattern("yyyy MM dd H m")) + " / "
+                + to.format(DateTimeFormatter.ofPattern("yyyy MM dd H m"));
```

Comment:
Great to see you continue your code in the next line to prevent going over 120 characters per line